README.md

```diff
@@ -33,6 +33,29 @@ A file named `{NODE_ENV}.json` can also be placed alongside the `default.json`,
 
 For more information see the [documentation](https://github.com/lorenwest/node-config/wiki/Environment-Variables#node_env).
 
+## Docker
+
+You can run these files on docker by first building the containers:
+```bash
+cd packages
+docker build -f DockerMicroservice -t openactive.microservice:latest ./
+docker build -f DockerIntegration -t openactive.integration:latest ./
```

Comment:
Just looking at this, is there any reason why we couldn't use a single dockerfile?

`npm run start-broker` and `npm run start-tests` can run in the root directory?

Also `npm start` runs both together and kills the mircoservice upon test completion... so this would allow users of the dockerfile to have this ability

It would also remove the need to copy test-interface-criteria, and means that `npm install` in the root (which installs all packages) would also work

Reply:
Took me ages to work out why I couldn't get this to work, turns out it was a node versioning issue. I've also moved to alpine containers so we should be able to look into publishing a container on docker hub so people can just do `docker run --rm openactive.test-suite` in future

Reply:
Great idea!